src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -66,7 +66,7 @@ internal partial class SolutionState
             _projectIds = projectIds.ToImmutableReadOnlyListOrEmpty();
             _projectIdToProjectStateMap = idToProjectStateMap;
             _projectIdToTrackerMap = projectIdToTrackerMap;
-            _linkedFilesMap = linkedFilesMap;
+            _filePathToDocumentIdsMap = filePathToDocumentIdsMap;
```

Comment:
Did we do an audit through existing callers of GetDocumentIdsForFilePath to see if this is going to crash anybody if they get new results? A lot of the places do actually do a GetDocument and handle nulls so I'm not terribly worried, but I had some hesitation going this direction if that was going to be a big problem.

Reply:
Yes, I did confirm so.